src/Scrolling.md

```diff
@@ -4,14 +4,17 @@
 These registers can be accessed even during Mode 3, but modifications may not take
 effect immediately (see further below).
 
-## FF42–FF43 — SCY, SCX: Viewport Y position, X position
+## FF42–FF43 — SCY, SCX: Background viewport Y position, X position
 
-Those specify the top-left coordinates of the visible 160×144 pixel area within the
+These two registers specify the top-left coordinates of the visible 160×144 pixel area within the
 256×256 pixels BG map. Values in the range 0–255 may be used.
 
+The PPU calculates the bottom-right coordinates of the viewport with those formulas: `bottom := (SCY + 143) & 255` and `right := (SCX + 159) & 255`.
+As we can infer, in case the values are larger than 255 they will wrap around towards the top-left corner of the screen.
+
 ## FF4A–FF4B — WY, WX: Window Y position, X position plus 7
 
-Specify the top-left coordinates of [the Window](#Window).
+These two registers specify the top-left coordinates of [the Window](#Window).
```

Comment:
Good catch that the change applied to here as well!

```suggestion
These two registers specify the on-screen coordinates of [the Window](#Window)'s top-left pixel.
```

I think this wording is clearer with regards to the coordinate space, and what the coordinates refer to?